makefile

```diff
@@ -183,14 +183,24 @@ DEBUG_SET_ENV_VARS := \
 	export FEATURE_EU_EXIT_FORMS_ENABLED=true; \
 	export EU_EXIT_ZENDESK_SUBDOMAIN=debug
 
+TEST_SET_ENV_VARS := \
+    export DIRECTORY_FORMS_API_BASE_URL=http://forms.trade.great:8011; \
+    export DIRECTORY_FORMS_API_API_KEY=debug; \
+	export DIRECTORY_FORMS_API_SENDER_ID=debug; \
```

Comment:
funky indentation

Reply:
Yeah, it's not in the file if you open it